.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
-## Pull Request Guidelines
+<!--
+Checklist for submitting a Pull Request
+
+Please remove this comment block before submitting.
 
 1. Please read our [contributor guidelines](https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md).
-1. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
-1. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below.
+2. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
+3. If you want this PR to automatically close an issue when it is merged,
+   add `fixes #<issue number>` or `fixes #<issue number>, fixes #<issue number>`
+   to close multiple issues (see: https://github.com/blog/1506-closing-issues-via-pull-requests).
+4. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below.
+-->
```

Comment:
Oh I see, that was intentional.  Ultimately this leaves the important block (with instructions) in there, but if the developer hurries past the initial PR screen the commented block will be invisible otherwise - unless the developer presses "Edit" on that comment.  That feels a bit odd, but as I write this I'm starting to like it.  Yes, I like it.  Let's see how it works for people.